src/main/java/edu/harvard/iq/dataverse/pidproviders/doi/crossref/CrossRefDOIPidProvider.java

```diff
@@ -0,0 +1,125 @@
+package edu.harvard.iq.dataverse.pidproviders.doi.crossref;
+
+import edu.harvard.iq.dataverse.DvObject;
+import edu.harvard.iq.dataverse.GlobalId;
+import edu.harvard.iq.dataverse.pidproviders.doi.AbstractDOIProvider;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class CrossRefDOIPidProvider extends AbstractDOIProvider {
```

Comment:
The other DOI providers drop the Pid part of the name (or use DOI instead of Pid since it is the type of Pid they support), so this would be more consistent as CrossRefDOIProvider.

Reply:
renaming